tests/integration/remote/util.py

```diff
@@ -8,13 +8,21 @@
 import time
 import warnings
 
+import web3.exceptions
```

Comment:
@mariacarmina we can keep the full import e.g. web3.exceptions.ExtraDataLengthError and use it directly inside ERRORS_TO_CATCH. Otherwise let's remove this import, because it is not used